security/citadel/templates/deployment.yaml

```diff
@@ -70,6 +70,12 @@ spec:
           env:
           - name: CITADEL_ENABLE_NAMESPACES_BY_DEFAULT
             value: "{{ .Values.security.enableNamespacesByDefault }}"
+          {{- if .Values.env }}
```

Comment:
SHouldn't it be `.Values.security.env`, have raised a followup PR to fix this: https://github.com/istio/installer/pull/423